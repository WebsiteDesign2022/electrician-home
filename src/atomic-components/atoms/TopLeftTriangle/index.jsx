import React from 'react';
import './index.scss';

const TopLeftTriangle = ({ color }) => (
    <div className={`arrow-top-left-${color}`}></div>
);

export default TopLeftTriangle;
