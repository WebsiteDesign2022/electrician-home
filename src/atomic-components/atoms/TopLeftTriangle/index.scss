

.arrow-top-left-black {
    width: 0; 
    height: 0; 
    border-top: 10px solid black;
    border-left: 10px solid black;
    border-bottom: 10px solid transparent;
    border-right:10px solid transparent;
}

.arrow-top-left-orange {
    width: 0; 
    height: 0; 
    border-top: 10px solid #FDB409;
    border-left: 10px solid #FDB409;
    border-bottom: 10px solid transparent;
    border-right:10px solid transparent;
}
