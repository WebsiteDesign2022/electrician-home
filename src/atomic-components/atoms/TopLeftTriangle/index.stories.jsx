import React from 'react';

import TopLeftTriangle from './index';

export default {
  title: 'Electrician-Home/atoms/TopLeftTriangle',
  component: TopLeftTriangle,
  parameters: {
    // More on Story layout: https://storybook.js.org/docs/react/configure/story-layout
    layout: 'fullscreen',
  },
};

const Template = (args) => <TopLeftTriangle {...args} />;

// More on args: https://storybook.js.org/docs/react/writing-stories/args
TopLeftTriangle.args = {
};

export const Primary = Template.bind({});