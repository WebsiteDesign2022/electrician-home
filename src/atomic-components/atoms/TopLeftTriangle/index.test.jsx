import TestRenderer from 'react-test-renderer';

import TopLeftTriangle from '.';

describe("TopLeftTriangle", () => {
    test("renders TopLeftTriangle", () => {
        const topLeftTriangle = TestRenderer.create(
            <TopLeftTriangle />
        );

        expect(topLeftTriangle.toJSON()).toMatchSnapshot();
    });
});
