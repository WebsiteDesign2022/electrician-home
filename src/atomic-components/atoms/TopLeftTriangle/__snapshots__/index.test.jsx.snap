// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopLeftTriangle renders TopLeftTriangle 1`] = `
<div
  className="arrow-top-left-undefined"
/>
`;
